packages/app/src/components/uiBlocks/GroupDropDownItem.vue

```diff
@@ -70,10 +72,13 @@ export default defineComponent({
       if (!activeItemIndex) {
         return;
       }
-
       activeItemIndex.value = isOpen.value ? null : itemIndex.value;
     };
 
+    watch(isOpen, (newVal) => {
```

Comment:
I would change this, so that you emit some `isOpen` event to the parent which then can set the `selectedTag` to `null`.